locales/en.default.schema.json

```diff
@@ -675,11 +681,18 @@
           "options__5": {
             "label": "Inverse"
           },
-          "label": "Color scheme"
+          "label": "Color scheme",
+          "info": "Visible when text box displayed"
```

Comment:
@gretchen-willenberg The image banner updates are not merged yet, but I don't see the text `Hidden when text box displayed` as raised in your audit, is it possible that this has changed already to another form?

cc. @Oliviammarcello